mne/viz/_brain/_timeviewer.py

```diff
@@ -397,17 +400,14 @@ def __init__(self, brain):
         self.playback_speed = default_playback_speed
         self.refresh_rate_ms = max(int(round(1000. / 60.)), 1)
         self.plotter.add_callback(self.play, self.refresh_rate_ms)
-        self.plotter.add_key_event('space', self.toggle_playback)
 
         # add toggle to show/hide interface
         self.visibility = True
-        self.plotter.add_key_event('y', self.toggle_interface)
 
-        # apply auto-scaling action
-        self.plotter.add_key_event('t', self.apply_auto_scaling)
-
-        # restore user scaling action
-        self.plotter.add_key_event('u', self.restore_user_scaling)
+        # display help
+        self.help_actor = self.plotter.add_text(
+            text="Press ? to show help window"
+        )
```

Comment:
I would rather not sacrifice this much (or really any) screen real estate for this:

![Screenshot from 2020-02-13 09-33-22](https://user-images.githubusercontent.com/2365790/74445206-02351f80-4e44-11ea-8082-c3c249b7a4ba.png)


If it's always `?` in all of our plotters we can just document it in a tutorial and in docstrings.

Reply:
... actually the intuitive way to do it without eating up screen real estate would be to add a `Help->Show MNE key bindings...` menu option at the top. If it's not too hard to do it it's a nice option

Reply:
Or more completely / standardly, `Help-> Show MNE key bindings...    ?` (with the `?` in the right column of the menu item to indicate it's the shortcut)

Reply:
Thanks for the suggestion @larsoner. I'm on it

Reply:
Just need to find this code snippet I did not so long ago...

Reply:
Found it: https://github.com/mne-tools/mne-python/pull/7200#issuecomment-574634662